commit 15a7ed94d94d06afe321ecb4849aeaf7162800d2
Author: Dongli Zhang <dongli.zhang@oracle.com>
Date:   Fri May 27 15:41:21 2016 +0800

    xen: add printk to grant table

diff --git a/xen/common/grant_table.c b/xen/common/grant_table.c
index 8b22299..b08a38b 100644
--- a/xen/common/grant_table.c
+++ b/xen/common/grant_table.c
@@ -3437,6 +3437,19 @@ void grant_table_init_vcpu(struct vcpu *v)
     v->maptrack_tail = MAPTRACK_TAIL;
 }
 
+void my_old_func(struct domain *rd)
+{
+	printk("We are in function my_old_func\n");
+	if (rd->domain_id == 0)
+		printk("message for dom 0\n");
+
+	if (rd->domain_id == 1)
+		printk("message for dom 1\n");
+
+	if (rd->domain_id > 1)
+		printk("message for other dom\n");
+}
+
 static void gnttab_usage_print(struct domain *rd)
 {
     int first = 1;
@@ -3446,6 +3459,8 @@ static void gnttab_usage_print(struct domain *rd)
     printk("      -------- active --------       -------- shared --------\n");
     printk("[ref] localdom mfn      pin          localdom gmfn     flags\n");
 
+	my_old_func(rd);
+
     grant_read_lock(gt);
 
     for ( ref = 0; ref != nr_grant_entries(gt); ref++ )
