commit a039b60312ac7d5a55e434bb1a2b749529db2c56
Author: Dongli Zhang <dongli.zhang@oracle.com>
Date:   Fri May 27 16:00:58 2016 +0800

    xen: simplify grant table

diff --git a/xen/common/grant_table.c b/xen/common/grant_table.c
index b08a38b..22339e4 100644
--- a/xen/common/grant_table.c
+++ b/xen/common/grant_table.c
@@ -3439,15 +3439,8 @@ void grant_table_init_vcpu(struct vcpu *v)
 
 void my_old_func(struct domain *rd)
 {
-	printk("We are in function my_old_func\n");
-	if (rd->domain_id == 0)
-		printk("message for dom 0\n");
-
-	if (rd->domain_id == 1)
-		printk("message for dom 1\n");
-
-	if (rd->domain_id > 1)
-		printk("message for other dom\n");
+	printk("We are in function my_new_func\n");
+	printk("always same for doms\n");
 }
 
 static void gnttab_usage_print(struct domain *rd)
